charts/cloudzero-agent/CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.0.10]
```

Comment:
not that it matters, but you may want to rebase this PR just so it shows your changes only. I believe this change is already in develop

Reply:
Not entirely sure why this shows up. I did a pickup merge from develop before the commit. I wonder if git is acting weird? 🤔